src/usr/local/containerbase/tools/v2/rye.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function install_tool () {
+  local versioned_tool_path
+  local arch
+  local name
+  local version
+  local platform
+  local repo
+
+  arch=${ARCHITECTURE}
+  name=$TOOL_NAME
+  version=$TOOL_VERSION
+  platform=$(uname -s)
+  repo="astral-sh/$name"
+
+  if [[ $platform == "Linux" ]]; then
+    platform="linux"
+  fi
+
+  if [[ $arch == armv8* ]] || [[ $arch == arm64* ]] || [[ $arch == aarch64* ]]; then
+    arch="aarch64"
+  elif [[ $arch == i686* ]]; then
+    arch="x86"
+  fi
+
+  binary="${name}-${arch}-${platform}"
+  file_name="${binary}.gz"
+  base_url="https://github.com/${repo}/releases/download/${version}"
+  file_url="${base_url}/${file_name}"
+
+  checksum_file=$(get_from_url "${file_url}.sha256")
+  # get checksum from file
+  expected_checksum=$(cut -d' ' -f1  "${checksum_file}")
+
+  file=$(get_from_url "${file_url}" "${file_name}" "${expected_checksum}" "sha256sum")
+
+  versioned_tool_path=$(create_versioned_tool_path)
+  create_folder "${versioned_tool_path}/bin"
+
+  gunzip -c "${file}" > "${TEMP_DIR}/rye"
```

Comment:
use bsdtar

Reply:
Can't use bsdtar: the file is not a `tar.gz`, just a plain `gz`